.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 2025
+
+jobs:
+  lint-prettier-fix:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: true
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run Prettier and fix formatting
+        run: |
+          npx prettier . --write
+
+      - name: Run ESLint and fix problems
+        run: |
+          npx eslint . --ext .js,.jsx,.ts,.tsx --fix
+
+      - name: Check if there are changes
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
```

Comment:
From where does it come from this email?

Reply:
I don't have an answer for it but I wouldn't sweat over the details of this part because everything is handled with autofix.ci